target/linux/ramips/image/mt76x8.mk

```diff
@@ -206,6 +206,14 @@ define Device/dlink_dap-1325-a1
 endef
 TARGET_DEVICES += dlink_dap-1325-a1
 
+define Device/dlink_dap-1610-a1
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_PACKAGES := kmod-mt76
```

Comment:
`kmod-mt76` does not contain a driver by itself, you should add the real driver here.